### PR TITLE
add overlays examples

### PR DESCRIPTION
**Pull request by aabedraba**
- fix invalid title
- add overlays examples

This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: examples/overlays/overlay.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+overlays: 1.0.0
+extends: ./openapi.yaml
```
Comment (jasmingacic) [inline examples/overlays/overlay.yaml:2]: do we have an explanation in our docs that extends must be in the form of absolute/relative path i.e. .openapi.yaml won't work but ./openapi.yaml will work?

I see it as a potential problem

**Review by jasmingacic — APPROVED** [review]
Just a small note
Otherwise it is good to go

File: examples/overlays/overlay.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+actions:
+  - target: "$"
+    update:
+      x-kusk: 
+        mocking:
+          enabled: true
```
Comment (kylehodgetts) [inline examples/overlays/overlay.yaml:8]: new line here

**Review by mbana — APPROVED** [review]
LGTM.

**Pull request merged**